Makefile

```diff
@@ -127,6 +127,15 @@ test:
 	cd provider/shim && go test -v .
 	cd examples && go test -v -tags=all -parallel $(TESTPARALLELISM) -timeout 2h
 
+# Tests that the work-in-progress provider causes no replace or update plans for stacks deployed
+# using the last released (baseline) version.
+test.upgrade::
```

Comment:
If this creates problems for ci-mgmt-ification I can drop for now.

Reply:
The current ci-mgmt PR doesn't handle the Makefile at all. We can merge changes and when we get aws to use a ci-mgmt generated Makefile we will accommodate then.